enzy_htp/structure/ligand.py

```diff
@@ -100,6 +100,22 @@ def add_conformer(self, points:List[Tuple[float,float,float]] ) -> int:
 
         return self.n_conformers()
 
+    def get_ligand_conformer(self, idx:int) -> Ligand:
```

Comment:
This will be merged into the a method of StructureEnsemble in the future FYI.
Probably add a TODO about this next to this func?

Reply:
Fixed this and added a TODO note about it.

Reply:
Nice